src/base/perf_monitor.h

```diff
@@ -347,16 +347,8 @@ typedef enum
 
   PSTAT_BT_ONLINE_NUM_INSERTS,
   PSTAT_BT_ONLINE_NUM_INSERTS_SAME_PAGE_HOLD,
-  PSTAT_BT_ONLINE_NUM_REJECT_NO_MORE_KEYS,
-  PSTAT_BT_ONLINE_NUM_REJECT_MAX_KEY_LEN,
-  PSTAT_BT_ONLINE_NUM_REJECT_NO_SPACE,
-  PSTAT_BT_ONLINE_NUM_RELEASE_LATCH,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_NOT_IN_RANGE1,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_NOT_IN_RANGE2,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_NOT_IN_RANGE3,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_NOT_IN_RANGE4,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_FALSE_FAILED_RANGE1,
-  PSTAT_BT_ONLINE_NUM_REJECT_KEY_FALSE_FAILED_RANGE2,
+  PSTAT_BT_ONLINE_NUM_RETRY,
+  PSTAT_BT_ONLINE_NUM_RETRY_NICE,
 
   PSTAT_BT_ONLINE,
```

Comment:
Please move PSTAT_BT_ONLINE first of its group and give a more substantial name, e.g. "btree_online_load[ing]" or "btree_online_build".